plugins/module_utils/openvpn_server.py

```diff
@@ -316,6 +313,13 @@ def _create_target(self):
     def _find_target(self):
         """ find the XML target_elt """
         (target_elt, self.idx) = self._find_openvpn_server(self.obj['description'])
+        for param in ['shared_key', 'tls']:
+            before_value = self.pfsense.get_element(param, target_elt)
+            if before_value is None and self.params[param] == 'generate':
+                (_, key, stderr) = self.module.run_command('/usr/local/sbin/openvpn --genkey secret /dev/stdout')
```

Comment:
linter say _ is bad -> how should an unused variable be named?

Reply:
i name it `unused` now: https://peps.python.org/pep-0640/

Reply:
FWIW - that PEP was rejected.  We are using `dummy` elsewhere in the code, so let's stick with that.